src/main/webapp/i18n/de/exampleSubmission.json

```diff
@@ -5,6 +5,17 @@
                 "title": "Beispielabgabe-Board",
                 "editor": "Beispielabgabe-Editor"
             },
+            "feedback": {
+                "CORRECT": "Richtiges Feedback ✅",
+                "INCORRECT_SCORE": "Falsche Punktzahl ❌",
+                "UNNECESSARY_FEEDBACK": "Redundantes Feedback ❌",
+                "MISSING_GRADING_INSTRUCTION": "Fehlende Bewertungsanweisung ❌",
+                "INCORRECT_GRADING_INSTRUCTION": "Falsche Bewertungsanweisung ❌"
+            },
+            "submissionValidation": {
+                "missing": "Nicht genügend Feedbacks. Die Anzahl der Feedbacks sollte die gleiche sein wie die der Instruktoren",
```

Comment:
Feedbacks does not exist in German. You could write: "Zu wenig Feedback gegeben. In der Musterbewertung wurde an mehr Stellen Feedback gegeben."
-> you should also avoid "Instruktoren" (generic plural masculin) and I suggest using Musterbewertung (Example Submission) to avoid using "Instruktor:innen". But you could also use Instruktor:innen.